inventory-generation/notifications/main.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Generate email notifications to be consumed by Ansible Tower scheduled jobs
+  hosts: local
+  gather_facts: false
+  
+  tasks:
+    - name: "Fail If Working Directory Is Not Provided"
+      fail:
+        msg: "directory var needs to be provided in order to ensure a safe working directory"
```

Comment:
Please update the message to be reflecting the checks it does - i.e.: `directory and email_templates_directory` - or otherwise just more generic to say something `required data sources / directories not defined`

Reply:
Thanks for catching that, updated message to be more descriptive.